CorvallisBus.Test/TransitManagerTests_ScheduleMerging.cs

```diff
@@ -732,12 +732,12 @@ public void EstimatesAreForPreviousDay_WhenAdjacentDays_AndEarlyMorningWithOvern
             mockRepo.Setup(repo => repo.GetPlatformTagsAsync()).Returns(Task.FromResult(new Dictionary<int, int> { { 12345, 123 } }));
 
             var testEstimate = new ConnexionzPlatformET(
-                platformTag: 123,
-                routeEstimatedArrivals: new List<ConnexionzRouteET>
+                PlatformTag: 123,
```

Comment:
If I were to do this refactoring again, I would probably start by deleting the constructor body content, then renaming all the parameters, then changing the type to a positional record.